src/modules/transaction/utils.spec.ts

```diff
@@ -194,15 +194,13 @@ describe('modules', function () {
           expect(getTransactionHashFromAction(action)).toEqual(tx.hash)
         })
 
-        it('should throw for a normal action (use isTransactionAction to avoid it)', function () {
+        it('should throw for a normal action (use isTransactionAction to avoid it)', function() {
           const action = {
             type: '[Success] Transaction action',
             payload: { more: 2 }
           }
 
-          expect(() => getTransactionHashFromAction(action)).toThrowError(
-            "Cannot read property 'hash' of undefined"
-          )
+          expect(() => getTransactionHashFromAction(action)).toThrow
```

Comment:
toThrow should be a function call:
```suggestion
          expect(() => getTransactionHashFromAction(action)).toThrow()
```
Was it changed because the error changed?

Reply:
Yes, the error message changed between node versions